cruise-control/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this isn't necessary, let's remove this. If you need it locally then please don't add it to the change.